openssl/sha_test.go

```diff
@@ -65,3 +65,19 @@ func TestSha(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkHash8Bytes(b *testing.B) {
+	b.StopTimer()
+	h := NewSHA256()
+	sum := make([]byte, h.Size())
+	var buf = make([]byte, 8192)
```

Comment:
Could this use short assignment for all the vars?

```suggestion
	sum := make([]byte, h.Size())
	buf := make([]byte, 8192)
```